modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/portlet/action/DeleteStyleBookEntryPreviewMVCActionCommand.java

```diff
@@ -45,11 +45,11 @@ protected void doProcessAction(
 			_styleBookEntryLocalService.fetchStyleBookEntry(styleBookEntryId);
 
 		if (styleBookEntry != null) {
-			PortletFileRepositoryUtil.deletePortletFileEntry(
-				styleBookEntry.getPreviewFileEntryId());
-
 			_styleBookEntryService.updatePreviewFileEntryId(
 				styleBookEntryId, 0);
+
+			PortletFileRepositoryUtil.deletePortletFileEntry(
```

Comment:
@thiago-buarqque is there a reason why we don't delete the file in stylebookEntryLocalServiceImpl instead?